lib/pretty_text.rb

```diff
@@ -230,20 +230,22 @@ def self.unescape_emoji(title)
 
     set = SiteSetting.emoji_set.inspect
     custom = Emoji.custom.map { |e| [e.name, e.url] }.to_h.to_json
+    inline_emoji = SiteSetting.enable_inline_emoji_translation
     protect do
       v8.eval(<<~JS)
         __paths = #{paths_json};
-        __performEmojiUnescape(#{title.inspect}, { getURL: __getURL, emojiSet: #{set}, customEmoji: #{custom} });
+        __performEmojiUnescape(#{title.inspect}, { getURL: __getURL, emojiSet: #{set}, customEmoji: #{custom}, inlineEmojiEnabled: #{inline_emoji} });
```

Comment:
I think splitting this line is easier on the 👀 

```suggestion
        __performEmojiUnescape(#{title.inspect}, { 
          getURL: __getURL, 
          emojiSet: #{set}, 
          customEmoji: #{custom}, 
          inlineEmojiEnabled: #{inline_emoji} 
        });
```